galactus.info.yml

```diff
@@ -12,7 +12,7 @@ libraries-override:
   classy/base: false
   #core/modernizr: false
 
-ckeditor_stylesheets:
+ckeditor5_stylesheets:
```

Comment:
Can you leave the ckeditor4 version in place as well so this will work for either case?

Reply:
Should I have the following?
`
ckeditor_stylesheets:
ckeditor5_stylesheets:
  - css/ckeditor-iframe.css
`

Reply:
I don't think you can do that in YAML, and I suggest a different filename for the CSS/SASS file for the ckeditor5 in a different comment

Reply:
I don't quite understand what you meant. Could you explain it in more details?

Reply:
In YAML, I don't believe you can put two keys and assign them both the same value.
```
ckeditor_stylesheets:
ckeditor5_stylesheets:

    css/ckeditor-iframe.css
```
Is interpreted as:
```
ckeditor_stylesheets: null
ckeditor5_stylesheets: css/ckeditor-iframe.css
```

In the [sass/ckeditor-iframe.scss](https://github.com/ubc-web-services/galactus/pull/62/files/f2adffb83b06760f1f9cb6e940307ceda6f5c587#diff-cc4543baca2dc4631899f8c2f8cd4835e800d756b6b0174c74cb39a8c6a51f82) comment below I suggest having a different file with the changes for ckeditor5 `ckeditor5-iframe.scss ` that way ckeditor4 people can keep using that